### ISSUE TITLE
bug(Datepicker): The calendarLabel property is not read by the screen reader, and is not assigned in the DOM

### ISSUE THREAD

**Issue by lidiagabriela** [angular/components#27935]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

The calendarLabel property is not read by the screen reader and is not assigned in the DOM.
I also searched in the source code of the calendar component and it seems that it exists only in the documentation, and in the html or ts code it is not assigned anywhere.

### Reproduction

StackBlitz link: https://stackblitz.com/edit/lhj2eb?file=src%2Fapp%2Fmy-date-picker-intl.service.ts
Steps to reproduce:
1. Open stackblitz example
2. Open NVDA screen reader (or other screen reader)
3. Focus the datepicker icon
4. Press 'enter' key to open the calendar popup

### Expected Behavior

The calendarLabel property should be read by the screen reader and should exist in the DOM.

### Actual Behavior

The calendarLabel property is not read by the screen reader and is not assigned in the DOM.

### Environment

- Angular: v15.0.4/v15.2.9
- CDK/Material: v15.2.9
- Browser(s): doesn't matter (I tested on Microsoft Edge version 117.0.2045.60)
- Operating System (e.g. Windows, macOS, Ubuntu): Windows 10 Enterprise v21H2

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
